Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -228,10 +228,15 @@ public float pollingFrequency
         public float unscaledGameTime => Time.unscaledTime;
 
         public bool runInBackground => Application.runInBackground ||
-        // certain platforms ignore the runInBackground flag and always run. Make sure we're
-        // not running on one of those.
+        // certain platforms ignore the runInBackground flag and pump updates even if it is set to false. If we're
+        // running on one of those, just force the runInBackground flag to true. If we ever add a native flag like
+        // ignoresRunInBackground or something similar, we can check that here and remove the specific platform checks.
+        // https://issuetracker.unity3d.com/issues/ps5-inputsystem-devices-are-not-removed-properly-when-pad-turned-off-with-application-in-background
         // TODO: Add more platforms here as they're discovered.
-        Application.platform == RuntimePlatform.PS5;
+        Application.platform == RuntimePlatform.PS5 ||
+        Application.platform == RuntimePlatform.PS4 ||
+        Application.platform == RuntimePlatform.GameCoreXboxOne ||
+        Application.platform == RuntimePlatform.GameCoreXboxSeries;
```

Comment:
Nice for adding all these console platforms 👍 
Just to have more clarity on this: 
- Did you (or someone) validate on all of these platforms?
- Also, the ticket talks mostly about PS5 and package version 1.3.0. I assume this was probably already fixed in 1.5.0 for PS5, as it was already a platform that run in the background before this PR?
- This PR only adds the PS4 and Xbox console platform checks, which are also commented on the UUM issue, right?